test/integration/zk-discovery.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load helpers
+
+ZK_HOST=127.0.0.1:2181
```

Comment:
Instead of hardcoding the port, you could do something similar to:
https://github.com/chanwit/swarm/blob/test-zk-discovery/test/integration/helpers.bash#L11

Example:
```bash
# Address on which Zookeeper will listen (random port between 7000 and 8000).
ZK_HOST=127.0.0.1:$(( ( RANDOM % 1000 )  + 7000 ))
```